*** graph_input.c.421	Mon Apr 27 12:41:28 1998
--- graph_input.c	Wed Mar 18 13:41:38 1998
***************
*** 597,603 ****
          termvar.c_oflag = 0 ;
! /*       termvar.c_cc[VEOL] = 0 ;*/
! /*       termvar.c_cc[VEOF] = 0 ;*/
! /* changed for Linux 1/98*/
          termvar.c_cc[VTIME] = 0 ;
          termvar.c_cc[VMIN] = 0 ;
          termvar.c_lflag = 0 ;
--- 597,605 ----
          termvar.c_oflag = 0 ;
! #if defined(VMIN) && defined(VTIME)
          termvar.c_cc[VTIME] = 0 ;
          termvar.c_cc[VMIN] = 0 ;
+ #else
+         termvar.c_cc[VEOF] = 0 ;
+         termvar.c_cc[VEOL] = 0 ;
+ #endif
          termvar.c_lflag = 0 ;
